python/buffer_autoclose.py

```diff
@@ -66,6 +69,10 @@ def get_all_buffers():
     infolist = w.infolist_get('buffer', '', '')
     while w.infolist_next(infolist):
         buffer_type = w.buffer_get_string(w.infolist_pointer(infolist, 'pointer'), 'localvar_type')
+        name = w.buffer_get_string(w.infolist_pointer(infolist, 'pointer'), 'name')
+        if name in w.config_get_plugin('prefer').split(','):
```

Comment:
For optimization reason, you should perform this split before the while loop, so it's done only once (as the value of option does not change during this loop).

Reply:
Makes sense, thanks for noticing. :-)